### PR TITLE
Docker is here!!

### PR DESCRIPTION
**Pull request by gorkalaucirica**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | no
| License       | MIT
| Doc PR        | no

Added docker with the following containers:
* Nginx
* MySQL
* PHP7-FPM
* Elastic-Logstash-Kibana (ELK)
* Redis
* RabbitMQ
* Data volumes (for db and app data)

### CODE REVIEW

File: etc/docker/.env.dist [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Symfony application's path (absolute or relative)
```
Comment (benatespina) [inline etc/docker/.env.dist]: Header licenses in all the files ;)

File: etc/docker/.env.dist [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+SYMFONY_APP_PATH=./../../../kreta
+SYMFONY_ENV=dev
+
+# MySQL
+MYSQL_ROOT_PASSWORD=root
+MYSQL_DATABASE=kreta
```
Comment (benatespina) [inline etc/docker/.env.dist]: I don't understand this directive, we haven't got any kreta database

Comment (benatespina) [inline etc/docker/.env.dist]: ok, but one thing is a MySQL connection and other thing is the database itself. We need a kreta database too?

File: etc/docker/.env.dist (hunk 1 above)
Comment (gorkalaucirica) [inline etc/docker/.env.dist]: Added!

File: etc/docker/.env.dist (hunk 2 above)
Comment (gorkalaucirica) [inline etc/docker/.env.dist]: No, maybe we can leave just the root password until we can add multiple DBs

**Pull request merged**